mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx

```diff
@@ -20,13 +21,23 @@ export const TagStats = (props: TagStatsProps) => {
   return props.stats.map((tagStat) => (
     <Fragment key={tagStat.name}>
       <StatisticLine
-        description={`${tagStat.name} jobs finished:`}
+        description={<JobTag
+          key={tagStat.name}
```

Comment:
Why do we need key here?

Reply:
ok , check it

Reply:
In this context, this is done as a warning from react: Each child in a list should have a unique "key" prop